pymbolic/mapper/equalizer.py

```diff
@@ -0,0 +1,169 @@
+import pymbolic
+
+
+class EqualizerMapper(pymbolic.mapper.Mapper):
```

Comment:
Many of these could be unified by simply using `init_arg_names` (as `__eq__` does currently).